### DISCUSSION TITLE
How can the Koreader read the book from the microSDXC of Android tablet?

### DISCUSSION THREAD

**Discussion by Dave616** [koreader/koreader#12202] [Q&A] [answered]
I installed the Koreader to my Samsung tablet, but it cannot read the PDF file which is at the microSDXC. 
Can any one help me for this issue ? Please.

Best,
Dave

**Comment by benoit-pierre** [comment]
Please provide more info: what do you mean when you say "it cannot read the PDF"? What's the error? Please provide the version of koreader, and more details about the steps leading to your issue.

**Reply by Dave616** [reply]
Sorry. The problem is resolved. It's seemly the problem of my tablet. When I click the name of book at Koreader, the Koreader did not response the click for a while. And the Koreader suddenly open the PDF file.  I do know what happened. But it does works now. 
Thanks for your reply. 

Best,
Dave

**Reply by benoit-pierre** [reply, marked as answer]
Yep, speed varies depending on the PDF contents (e.g. big images can slow down processing a lot), especially with reflow.
